src/main/java/com/lob/protocol/request/AbstractDataFieldRequest.java

```diff
@@ -5,35 +5,35 @@
 import java.util.Map;
 
 public abstract class AbstractDataFieldRequest extends AbstractLobRequest {
-    private final Map<String, String> data;
+    private final Map<String, String> mergeVariables;
 
     public AbstractDataFieldRequest(
             final Map<String, String> metadata,
-            final Map<String, String> data,
+            final Map<String, String> mergeVariables,
             final String description) {
         super(metadata, description);
-        this.data = data;
+        this.mergeVariables = mergeVariables;
     }
 
     public Map<String, String> getData() {
-        return data;
+        return mergeVariables;
     }
 
     @Override
     protected LobParamsBuilder beginParams() {
-        return super.beginParams().putMap("data", this.data);
+        return super.beginParams().putMap("mergeVariables", this.mergeVariables);
```

Comment:
This is what is building the request payload to make a request to Lob's API. The new parameter value is "merge_variables". We use camelCase naming conventions for variables but the actual key in the request should be `"merge_variables"`.

```java
return super.beginParams().putMap("merge_variables", this.mergeVariables);
```